### DISCUSSION TITLE
binlog同步数据异常

### DISCUSSION THREAD

**Discussion by yclxiao** [apache/doris#14105] [A - General / Q&A]
binlog同步数据到doris，任务的异常信息如下，请问如何解决
SyncFailMsg [type=SUBMIT_FAIL, msg=errCode = 2, detailMessage = execute batch failed, batchId: 4 ,msg: encounter exception when submit in channel 10033, table: cloud_uc_company, batch: 4]

**Comment by LemonLiTree** [comment]
doris哪个版本？

**Reply by yclxiao** [reply]
1.1.2
